conda/recipes/libraft_nn/meta.yaml

```diff
@@ -17,10 +17,6 @@ source:
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
   string: cuda{{ cuda_major }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
-  script_env:
```

Comment:
This line needs to be added back. We need the line `script_env:` to fix the problem @cjnolet noted.